### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by RobertAKARobin**
Tidied stylesheets, Javascripts, and views. Removed unused code, consolidated related code into files, etc.

### CODE REVIEW

File: app/assets/javascripts/submission.js [hunk 1]
```diff
@@ -1,24 +0,0 @@
-$(".js-score-update-form").on("submit", submitScore);
-$(".js-score-update-form input").on("focusout", submitScore);
```
Comment (adambray) [inline app/assets/javascripts/submission.js]: Just confirming that this is now covered by the `[data-autosave]` selector and functionality, correct?

File: app/views/memberships/show.html.erb [hunk 2]
```diff
@@ -33,9 +33,9 @@
   <section>
     <h3 class="fold" id="attendances"><a href="#attendances"><%= @user.first_name %>'s Attendance</a></h3>
     <p>Present: <%= @attendances.present.count %>, Tardy: <%= @attendances.tardy.count %>, Absent: <%= @attendances.absent.count %></p>
-    <div>
-        <%= render partial: "attendances/table", locals: {attendances: @attendances} %>
-    </div>
+    <table>
+      <%= render partial: "components/todo_table", locals: {collection: @attendances} %>
```
Comment (adambray) [inline app/views/memberships/show.html.erb]: :+1:

**Comment by adambray** [comment]
:+1:

**Comment by adambray** [comment]
radical

File: app/assets/javascripts/submission.js (hunk 1 above)
Comment (RobertAKARobin) [inline app/assets/javascripts/submission.js]: Yup!

**Pull request merged**